src/server.js

```
@@ -249,6 +250,48 @@ class Server {
     };
+
+    handleWebhook = async (req, res) => {
+        let event;
+        try {
+            event = this.components.stripe.validateEvent(req);
+        } catch (error) {
+            return res.status(400).send(`Webhook Error: ${err.message}`);
+        }
+        switch (event.type) {
+            case "payment_intent.succeeded":
+                const paymentIntent = event.data.object;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)